tensorflow/src/commonMain/kotlin/dev/icerock/moko/tensorflow/TensorDataType.kt

```diff
@@ -9,12 +9,20 @@ enum class TensorDataType(val value: Int) {
     FLOAT32(1),
     INT32(2),
     UINT8(3),
-    INT64(4);
+    INT64(4),
+    STRING(5),
+    BOOL(6),
+    INT16(7),
+    INT8(9);
 
     fun byteSize(): Int = when (this) {
         FLOAT32 -> 4
         INT32 -> 4
+        INT16 -> 2
+        INT8 -> 1
         UINT8 -> 1
         INT64 -> 8
+        BOOL -> -1
+        STRING -> -1
```

Comment:
how it can be? -1 bytes?

Reply:
It's copied from org.tensorflow.lite
![image](https://github.com/icerockdev/moko-tensorflow/assets/57789105/a90f9dd5-0136-4247-80f8-5febf237ba36)

Reply:
Byte size not declared in docs either [LINK](https://www.tensorflow.org/lite/api_docs/java/org/tensorflow/lite/DataType)